gulpfile.js

```diff
@@ -74,9 +72,38 @@ gulp.task('watch', function() {
 	  gulp.watch('app/*', ['deploy-local']);
 });
 
-gulp.task('deploy-local', ['build'], function() {	
-  return gulp.src(['dist/**/*', '!*.zip'])
-    .pipe(gulp.dest(LOCAL_OWA_FOLDER + '\\' + THIS_APP_ID));
+gulp.task('deploy-local', ['build'], function() {
+    var config;
+    var localOwaFolder;
+
+	try {
+	//Will throw exception when there is no config.json file
+	config = require('./config.json');
+    //Will throw exception when data in config.json is corrupted
+	localOwaFolder = config.LOCAL_OWA_FOLDER;
+	}
+	catch (err) {
```

Comment:
Could you please fix the formatting of the try block? It has some strange indentation.